test/integration/extensions/banking_config/extensions/banking.lua

```diff
@@ -0,0 +1,30 @@
+local checks = require('checks')
+-- local extensions = require('cartridge').service_get('extensions')
+
+local function customer_add(customer_id, fullname)
+    checks('number', 'string')
+    return box.space.customer:insert(
+        {customer_id, fullname}
+    )
+end
+
+local function account_add(customer_id, account_id, name)
+    checks('number', 'number', 'string')
+    return box.space.account:insert(
+        {customer_id, account_id, name, 0}
+    )
+end
+
+local function transfer_money(account_from, account_to, amount)
+    box.begin()
+        box.space.account:update({account_to}, {{'+', 4, amount}})
```

Comment:
Please, return back the normal indentation because I think that the current one is misleading.
You can leave blank lines between the start of the transaction and the end.